modules/oagw/oagw-gw/src/config.rs

```diff
@@ -0,0 +1,100 @@
+//! OAGW module configuration.
+
+use serde::Deserialize;
+
+/// OAGW module configuration.
+#[derive(Debug, Clone, Deserialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct OagwConfig {
+    // === Token Cache Settings (v3+) ===
+    /// Maximum capacity of the token cache.
+    pub token_cache_max_capacity: usize,
+    /// Maximum TTL for cached tokens in seconds.
+    pub token_cache_max_ttl_sec: u64,
```

Comment:
I think it should be std::time::Duration with humantime representation in yaml config.